benchmarks/BayesianInference/DiffEqBayesLotkaVolterra.jmd

```diff
@@ -67,6 +74,35 @@ The solution converges for tolerance values lower than 1e-3, lower tolerance lea
 @btime bayesian_result_stan = stan_inference(prob,t,data,priors,num_samples=10_000,print_summary=false, vars = (DiffEqBayes.StanODEData(), InverseGamma(2, 3)))
 ```
 
+### Direct Turing.jl
+
+```julia
+@model function fitlv(data, prob)
+    # Prior distributions.
+    σ ~ InverseGamma(2, 3)
+    α ~ truncated(Normal(1.5, 0.5), 0.5, 2.5)
+    β ~ truncated(Normal(1.2, 0.5), 0, 2)
+    γ ~ truncated(Normal(3.0, 0.5), 1, 4)
+    δ ~ truncated(Normal(1.0, 0.5), 0, 2)
+
+    # Simulate Lotka-Volterra model. 
+    p = SA[α, β, γ, δ]
+    _prob = remake(sprob, p = p)
+    predicted = solve(_prob, Tsit5(); saveat=0.1)
+
+    # Observations.
+    for i in 1:length(predicted)
+        data[:, i] ~ MvNormal(predicted[i], σ^2 * I)
+    end
+
+    return nothing
+end
+
+model = fitlv(odedata, prob)
+
+@btime chain = sample(model, NUTS(0.65), MCMCSerial(), 10000, 1; progress=false)
```

Comment:
I wonder if benchmark results change if the arguments are interpolated?

Reply:
I don't think so, compile time is minimal. People keep mentioning compile time stuff but if you measure it it's like ~0.3% IIRC, so it's a red herring. We should probably even just `@time` instead (and run it twice just to demonstrate it more clearly), but that's a change for another day.